src/traits.rs

```diff
@@ -0,0 +1,25 @@
+use super::Context;
+
+// https://doc.rust-lang.org/src/core/convert.rs.html#478-486
+pub trait TranslateFrom<T>: Sized {
```

Comment:
How is this different from the linked `TryFrom` trait? Just the name for documentation purposes?
What is it supposed to do? A short comment would help here.

Reply:
This is primarily to add context from the CLI.